bot/seasons/evergreen/fun.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import random
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Fun:
+    """
+    A collection of general commands for fun.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def roll(self, ctx, num_rolls: int):
```

Comment:
It would be good if there was a default value of 1. That way uses who don't present an argument get a plain single roll. Feels like something that would happen so it'll improve user experience.

```suggestion
    async def roll(self, ctx, num_rolls: int = 1):
```